src/index.js

```diff
@@ -134,8 +138,12 @@ class Dropzone extends React.Component {
   }
 
   open() {
+    const clickEvent = new MouseEvent('click', {
+      bubbles: false
+    });
+
     this.fileInputEl.value = null;
-    this.fileInputEl.click();
+    this.fileInputEl.dispatchEvent(clickEvent);
```

Comment:
Otherwise the click bubbles up and triggers the DropZone click event again. Example: https://jsfiddle.net/8c13nd2y/